clients/drcachesim/drpt2trace/ir2trace.cpp

```diff
@@ -126,8 +132,9 @@ ir2trace_t::convert(DR_PARAM_IN drir_t *drir,
             instr_t *next_instr = instr_get_next(instr);
             if (next_instr != nullptr) {
                 trace.push_back(
-                    { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_BRANCH_TARGET,
-                      reinterpret_cast<uintptr_t>(instr_get_app_pc(next_instr)) });
+                    { TRACE_TYPE_MARKER,
+                      TRACE_MARKER_TYPE_BRANCH_TARGET,
+                      { reinterpret_cast<uintptr_t>(instr_get_app_pc(next_instr)) } });
```

Comment:
Can't figure out what changed here?

Reply:
The braces around `reinterpret_cast<uintptr_t>(instr_get_app_pc(next_instr))`.